src/dynarec/rv64/dynarec_rv64_660f.c

```diff
@@ -2755,13 +2755,59 @@ uintptr_t dynarec64_660F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
             nextop = F8;
             GETEX(x2, 0);
             GETGD;
-            MV(gd, xZR);
-            for (int i = 0; i < 16; ++i) {
-                LB(x1, wback, fixedaddress + i);
-                SLT(x3, x1, xZR);
-                if (i > 0) SLLI(x3, x3, i);
-                OR(gd, gd, x3);
-            }
+            #define MYGO(s)            \
```

Comment:
Is this an optimisation?

Reply:
Yes. This instruction is computed in the reverse direction so we can make use of the SH1ADD instruction.